src/libs/scene_utils/README.md

```diff
@@ -0,0 +1,81 @@
+# IoT TwinMaker Scene SDK
+
+## Use
+
+This package provides sample command line scripts that automate the creation of IoT TwinMaker scenes. Upload 3D assets and place them in a 3D scene connected to IoT data via augmented widgets (tag, motion indicator, etc.).
+
+# Samples:
+
+## 1. COOKIE FACTORY `samples/cookie_factory_sample/`
+
+Use the models in the `scenes` directory for the `--assetDirPath` input.
+
+### Example
+
+If running this script from the root of `aws-iot-twinmaker-samples`:
+
+```
+npx ts-node src/libs/scene_utils/samples/cookie_factory_sample/sample.ts --workspaceId [WORKSPACE_ID] --sceneId [SCENE_ID] --assetDirPath src/workspaces/cookiefactory/scenes/
```

Comment:
may want to include full instructions in the sample to make it easier for customers: e.g. `npm install`